.github/workflows/repolinter.yml

```diff
@@ -25,7 +25,4 @@ jobs:
         uses: actions/checkout@v2
       - name: Run Repolinter
         if: ${{ steps.default-branch.outputs.result == 'true' }}
-        uses: newrelic/repolinter-action@v1
-        with:
-          config_url: https://raw.githubusercontent.com/newrelic/.github/main/repolinter-rulesets/community-plus.yml
-          output_type: issue
+        uses: newrelic/repolinter-action@v1
```

Comment:
nit : can You add newline at end of file please ?

Reply:
Absolutely.